cli/ballerina-cli/src/main/resources/new_cmd_defaults/manifest-app.toml

```diff
@@ -1,2 +1,12 @@
 [build-options]
 observabilityIncluded = true
+#offline = true
+#skipTests = false
+#cloud = "k8s"
+#taintCheck = true
+
+#[[platform.libraries]]
+#artifactId = "mysql-connector-java"
+#version = "5.1.45"
+#path = "./libs/mysql-connector-java-5.1.45.jar"
+#groupId = "mysql"
```

Comment:
@azinneera Are we going keep these commented like this?

Reply:
Yes, so that the user can uncomment and use easily.